components/application/activity-select.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import Link from 'next/link' // TODO: Should be avoided
```

Comment:
It ties us to Next.js. Even if we try to reimplement Link to avoid string dependency on Next, it makes us to import Link everywhere.

I would prefer to do it implicitly.

Reply:
I don't know. What's wrong with tieing us to Next.js? It was decided that we use Next.js. It's like we use bootstrap but we cannot import from it anything. Maybe I just don't understand. 

Link is a powerful component. It prefetches pages and does transition. Therefore I would not avoid it.

Reply:
This is a good point. I will consider it. Thank you!